HISTORY.rst

```diff
@@ -1,6 +1,19 @@
 History
 =======
 
+0.10.0 -- 2022-10-06
+-------------------
+
+New Features
+~~~~~~~~~~~~
+
+- Add support for preparing requests in BaseUrlSession
+
+Fixed Bugs
```

Comment:
Add doc bug fix: 

- Fix invalid syntax in session docs.  [commit](https://github.com/requests/toolbelt/commit/7c4f92bb81204d82ef01fb0f0ab6dba6c7afc075)
- Fix typo in docstring example [commit](https://github.com/requests/toolbelt/commit/8a4b8ea66b2b9cb5da42f414492e0218f62294b6)
- Fix MultipartEncoder docstring typo [commit](https://github.com/requests/toolbelt/commit/5ead53fdd220c4989081619043b2e5baa9712954)
- Fix typo in TCPKeepAliveAdapter [commit](https://github.com/requests/toolbelt/commit/50b52e487b533d1caa9e8b04ea31f6f75b07b2d6)

Reply:
Sounds good, I was just raising it just in case you thought it was a good thing to add.